performances/performance_test_msgs/msg/PerformanceHeader.msg

```diff
@@ -1,4 +1,5 @@
 builtin_interfaces/Time stamp
+string node_name
```

Comment:
is it a problem that the header is now not fixed in size, as a string has an arbitrary length? An other approach could be to save an generated ID (int or something else) based on the node_name.

Reply:
Good point.
Tools and shared memory implementations will now handle messages differently since all of them are now dynamic size.

Hashing the name may be a better solution.

I'm not sure whether IDL allows to have fixed size strings in the message... If it does, this could be another solution, with an arbitrary max node name length.

Reply:
Changed to uint64 and used item_name_to_id to hash the name to a unique id. I also performed a rebase to pull in the changes from master.